tests/ics/cal_no_dst.ics

```diff
@@ -4,17 +4,17 @@ PRODID:-//PIMUTILS.ORG//NONSGML khal / icalendar //EN
 BEGIN:VTIMEZONE
 TZID:America/Bogota
 BEGIN:STANDARD
-DTSTART;VALUE=DATE-TIME:19930403T230000
+DTSTART:19930206T230000
```

Comment:
For the record: this value changed due to changes in pytz: https://github.com/pimutils/khal/issues/1222

Reply:
Oh, should have read recent comments before commenting :swe